index.html

```diff
@@ -133,6 +133,51 @@ <h2 id="concepts">Learn about more advanced FFmpeg concepts</h2>
   </div>
   <!-- End Filtergraph explanation -->
 
+  <!-- Record and Live-Stream simultaneously -->
+    <span data-toggle="modal" data-target="#record-and-stream"><button type="button" class="btn btn-default" data-toggle="tooltip" data-placement="bottom" title="Record and Live-Stream simultaneously">Record and Live-Stream simultaneously</button></span>
+    <div id="record-and-stream" class="modal fade" tabindex="-1" role="dialog">
+      <div class="modal-dialog modal-lg">
+        <div class="modal-content">
+          <div class="well">
+            <h3>Record and Live-Stream simultaneously</h3>
+            <p><code>ffmpeg -re -i <i>${INPUTFILE}</i> -map 0 -flags +global_header -vf scale="1280:-1,format=yuv420p" -pix_fmt yuv420p -level 3.1 -vsync passthrough -crf 26 -g 50 -bufsize 3500k -maxrate 1800k -c:v libx264 -c:a aac -b:a 128000 -r:a 44100 -ac 2 -t ${STREAMDURATION} -f tee <i>"[movflags=+faststart]${TARGETFILE}|[f=flv]${STREAMTARGET}"</i></code></p>
+            I use this script to stream to a RTMP target and record the stream locally as .mp4 with only one ffmpeg-instance.<br />
+            As input, I use <code>bmdcapture</code> which is piped to ffmpeg. But it can also be used with a static videofile as input.<br />
+            The input will be scaled to 1280px width, maintaining height. Also the stream will stop after a given time (see <code>-t</code> option.)</p>
+            <p><strong>Note 1</strong><br />
```

Comment:
Rather than using a \<br \/\> tag here (and below with Note 2), I'd wrap the heading in a separate \<p\> or heading tag (maybe \<h3\>).  
You could also consider grouping the notes under a single heading.